gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/factories/GafferFactory.java

```diff
@@ -101,6 +104,32 @@ private static GafferSpec overrideGafferSpecConfig(final GafferSpec config, fina
         return opAuthoriser;
     }
 
+    private static ArrayList<Object> createOperationDeclaration(final GafferSpec config) {
+        ArrayList<Object> operations = new ArrayList<>();
+        if (config.getNestedObject(GAFFER_OPERATION_DECLARATION_KEY) != null) {
+            operations.add(config.getNestedObject(GAFFER_OPERATION_DECLARATION_KEY));
+        }
+        HashMap<String, Object> contents = new HashMap<>();
+        HashMap<String, String> contentsOfHandler = new HashMap<>();
```

Comment:
Maybe use better variable names than `contents` and `contentsOfHandler`, maybe like `proxyUrlDeclaration` and `proxyUrlClass` or something would be clearer.

Reply:
Updated